orm/models.py

```diff
@@ -49,11 +49,11 @@ def __new__(
 
 
 class QuerySet:
-    def __init__(self, model_cls=None, filter_clauses=None, select_related=None):
+    def __init__(self, model_cls=None, filter_clauses=None, select_related=None, limit_count=None):
         self.model_cls = model_cls
         self.filter_clauses = [] if filter_clauses is None else filter_clauses
         self._select_related = [] if select_related is None else select_related
-        self.limit_count = None
+        self.limit_count = None if limit_count is None else limit_count
```

Comment:
Just `self.limit_count = limit_count` is okay here. (filter_clauses and select_related are a little different since the empty case there is an empty list, not None)

Reply:
okay, I changed that, just initializing with 
```self.limit_count = limit_count```